main/options/options.cc

```diff
@@ -722,6 +725,7 @@ void readOptions(Options &opts,
         opts.lspSignatureHelpEnabled = enableAllLSPFeatures || raw["enable-experimental-lsp-signature-help"].as<bool>();
         opts.lspDocumentFormatRubyfmtEnabled =
             enableAllLSPFeatures || raw["enable-experimental-lsp-document-formatting-rubyfmt"].as<bool>();
+        opts.lspStaleStateEnabled = enableAllLSPFeatures || raw["enable-experimental-lsp-stale-state"].as<bool>();
```

Comment:
I have minor FUD about this, as I believe some developers at Stripe do run with all experimental features enabled as a matter of course?  I guess we can see if they become super-crashy if we start making use of this flag and can come up with a different implementation.

Reply:
Ah, thanks for calling that out! Although the flag is currently a no-op, I actually think we're going to want to reserve the right to keep this thing fairly crashy and stubby for at least a short while. For example, I'd like to start landing some of the threading-related stuff before namer and resolver updates are likely to be ready. So if folks are running with `enableAllLSPFeatures` in the real world, we probably _don't_ want to turn it on for them yet. So I propose just deleting `enableAllLSPFeatures || ` from the rhs here, meaning we'd require opt-in to this specific feature explicitly. Thoughts?

Reply:
I'm fine with just enabling this with the explicit, specific command-line flag so testing at Stripe is easier; we can subsume it under the all-features flag at a later point.